daemon/indexing/listener/listener.js

```diff
@@ -252,7 +305,7 @@ async function postToWebhook(urlString, json) {
   }
   return new Promise((resolve, reject) => {
     const req = http.request(postOptions, res => {
-      if(res.statusCode == 200){
```

Comment:
nit: in general when writing our JS code, shouldn't we always favor using the === vs the == operator ? Same comment for line 294 below.